osquery/events/kernel/circular_queue_user.cpp

```diff
@@ -76,12 +76,11 @@ osquery_event_t CQueue::dequeue(CQueue::event **event) {
   }
   header = (osquery_data_header_t *)read_;
   if (header->event != END_OF_BUFFER_EVENT) {
-    size_t size = osquery_sizeof_event(header->event)
-      + sizeof(osquery_data_header_t);
+    size_t size = header->size;
```

Comment:
Could just stick header->size into the next statement and removal the local size_t.